db/migrate/20230403113114_add_default_to_enabled.rb

```diff
@@ -0,0 +1,5 @@
+class AddDefaultToEnabled < ActiveRecord::Migration[7.0]
+  def change
```

Comment:
do we have any other way to set the default values other than passing from..to?

Reply:
we can do change_column_default :products, :enabled, false
but this will be irreversable